routing/payment_session.go

```diff
@@ -111,6 +117,10 @@ type paymentSession struct {
 	pathFindingConfig PathFindingConfig
 
 	missionControl MissionController
+
+	// minShardMaxAmt is the amount beyond which we won't try to further
+	// split the payment if no route is found.
```

Comment:
comment should note that this isn't applied when there is only one shard

Reply:
added